dlrm_s_pytorch.py

```diff
@@ -91,45 +97,8 @@
 # import torch.nn.functional as Functional
 # from torch.nn.parameter import Parameter
 
-from torch.optim.lr_scheduler import _LRScheduler
-
 exc = getattr(builtins, "IOError", "FileNotFoundError")
 
-class LRPolicyScheduler(_LRScheduler):
```

Comment:
Can we please keep these LR scheduler related changes?

Reply:
Thanks for the suggestion. I have added back the LRScheduler in the newest commit.